data-explorer/kusto/query/has-anyoperator.md

```diff
@@ -25,6 +25,9 @@ For more information about other operators and to determine which operator is mo
 | *col* | string | &check; | The column by which to filter.|
 | *expression* | scalar or tabular | &check; | An expression that specifies the values for which to search. Each expression can be a [scalar](scalar-data-types/index.md) value or a [tabular expression](tabularexpressionstatements.md) that produces a set of values. If a tabular expression has multiple columns, the first column is used. The search will consider up to 10,000 distinct values.|
 
+> [!WARNING]
+> For optimal performance, please stay within 2048 values. From 2049 values onwards, has_any() internally rewrites itself as regex.
```

Comment:
Note: This part "has_any() internally rewrites itself as regex." is in Alexander Sloutsky's words from here: https://stackoverflow.com/questions/59506801/has-any-performance-tanks-after-64-values

The limit here is 2048 because of the latest testing, this is the new limit, not 64 (as reported above).

Reply:
Feel free to take the "why" out, but please leave the guidance in. I've dealt with many teams who have run into this limit over the years and its a lot of headache and they wonder how come I know about this, and its not documented.

Reply:
Hi Basel, Alex suggested that we not make any docs changes, as these are implementation details that change from time to time and are data dependent. I'm going to close your PR. Thank you for the suggestion.